src/firefly/js/core/background/JobHistory.jsx

```diff
@@ -339,6 +350,13 @@ function convertToTableModel(jobs, tbl_id) {
     return table;
 }
 
+function makeEnumVals({columns, data, cname}) {
+    const cidx = columns.findIndex((c) => c.name === cname);
+    const vals = data.map((rowData) => rowData[cidx]);
+    columns[cidx].enumVals = uniq(vals.filter((d) => d)).join(',');
```

Comment:
You might want to put this in `TableUtil.js`, such as `makeClientTableEnumVals()`?

Reply:
I went ahead and apply logic to all client tables.  Don't forget to approve my `suit` PR. https://github.com/lsst/suit/pull/58